sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/AmqpConnectionScope.cs

```diff
@@ -1176,6 +1123,71 @@ protected virtual void CloseConnection(AmqpConnection connection)
             }
         }
 
+        /// <summary>
+        ///   Performs the actions needed to open an AMQP object, such
+        ///   as a session or link for use.
+        /// </summary>
+        ///
+        /// <param name="target">The target AMQP object to open.</param>
+        /// <param name="timeout">The timeout to apply when opening the link.  If the <paramref name="cancellationToken" /> is also passed, it will take precedence.</param>
+        /// <param name="cancellationToken">An optional <see cref="CancellationToken"/> instance to signal the request to cancel the operation.</param>
+        ///
+        /// <remarks>
+        ///     If both the <paramref name="timeout" /> and <paramref name="cancellationToken" /> are passed,
+        ///     the cancellation token will take precedence.
+        /// </remarks>
+        ///
+        protected virtual async Task OpenAmqpObjectAsync(AmqpObject target,
```

Comment:
Do we ever actually pass the timeout anymore?

Reply:
For the connection, yeah.